### ISSUE TITLE
Create a baseline

### ISSUE THREAD

**Issue by JasonLo** [UW-Madison-DSI/ask-xDD#2]
Using [covid-QA](https://github.com/deepset-ai/COVID-QA)

We set aside 20% data as test set and use it for benchmarking, the split is created by:

```python
from askem.data import COVID_QA
test_data = COVID_QA.train_test_split(test_size=0.2, seed=2023)["test"]
```

- [x] Chat-GPT bench
- [x] Petal.org bench
